tensorflow/lite/kernels/internal/reference/reference_ops.h

```diff
@@ -250,9 +251,8 @@ inline void ReluX(const tflite::ActivationParams& params,
   const T min_value = params.quantized_activation_min;
   for (int i = 0; i < flat_size; ++i) {
     const T val = input_data[i];
-    const T clamped = val > max_value   ? max_value
-                      : val < min_value ? min_value
-                                        : val;
+    const T clamped =
+        val > max_value ? max_value : val < min_value ? min_value : val;
```

Comment:
just confirming that this was a result of

```
clang-format -style=google
```

Reply:
yes